### DISCUSSION TITLE
Biometric ZKT comm password

### DISCUSSION THREAD

**Discussion by mrzeroo** [horilla-opensource/horilla#303] [Q&A]
My ZKT machines all have a comm password, Could we connect to it through horilla? Could you add a password text field below comm port so that we can complete the connection process?

**Comment by horilla-opensource** [comment]
> My ZKT machines all have a comm password, Could we connect to it through horilla? Could you add a password text field below comm port so that we can complete the connection process?

Hi @mrzeroo ,
We'll update the section and provide you with the code.

**Comment by mrzeroo** [comment]
Is there any updates please :-)

**Reply by mrzeroo** [reply]
Is there any updates please :-)

**Reply by horilla-opensource** [reply]
Hi @mrzeroo ,

Apologies for the delay.

The feature is now added to the system. Please check and let us know if you have any questions or concerns.
[[ADD] BIOMETRIC: Add zk_password field for BiometricDevice model](https://github.com/horilla-opensource/horilla/commit/adaf6ba09768fd3a723e01e1bac86db4a8fec473)
